src/stores/Store.ts

```diff
@@ -34,7 +34,7 @@ export default abstract class Store<TState> implements ActionReceiver, NotifySta
 
     abstract reduce(state: TState, action: Action): TState
 
-    private setState(value: TState) {
+    public setState(value: TState) {
```

Comment:
I don't think we want to have this public.

Reply:
Ok yes it's no longer needed since I use an action to set the initial state now.